mathesar_ui/src/systems/table-view/row/RowCell.svelte

```diff
@@ -58,6 +63,7 @@
   $: columnId = column.id;
   $: ({ activeCell, selectedCells } = selection);
   $: isActive = $activeCell && isCellActive($activeCell, row, processedColumn);
+  $: ({ 1: rId } = row.record);
```

Comment:
We have to pass the row id to the RowContextOptions and we were not using it directly in row cell anywhere so I destructured the record that is being sent and made this to send the row id to the RowContextOptions.
This can be directly destructured but would look a bit messy.
_<RowContextOptions recordId={Number(rId)} {recordsData} {row} />_

Reply:
You're obtaining the property `1` here, which is incorrect. You'll have to find the primary key column and use the row's value for it.

Reply:
Property `1` here is giving us the primary key of that row from the primary key column '`id`' and it cannot be changed nor removed and is must whenever we make a new row. 
How else do you want me to get that id of a row? I am unable to understand that, can you please guide...